tests/unit/InMemContainerTest.cpp

```diff
@@ -0,0 +1,68 @@
+#include <cppunit/extensions/HelperMacros.h>
+#include <GenSync/Data/InMemContainer.h>
+#include "InMemContainerTest.h"
+
+CPPUNIT_TEST_SUITE_REGISTRATION(InMemContainerTest);
+
+void InMemContainerTest::setUp(){
+    srand(1029);
+}
+
+void InMemContainerTest::functionalityTest() {
+    //Use a list to compare to the container, as in memory container should function exactly the same.
```

Comment:
I can't make sense of this ... each test method should have a javadoc description of the test.